<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>411</width>
    <height>482</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLineEdit" name="Series_Entry">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>40</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Series_Label">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>50</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Series Name:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="Add_Series_Button">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>420</y>
     <width>101</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Add Series</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Format_ComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>160</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>T20I</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>ODI</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Test</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Format_Label">
   <property name="geometry">
    <rect>
     <x>90</x>
     <y>170</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Format:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Category_ComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>100</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>Mens</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Womens</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="Category_Label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Category:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Team_1_ComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>220</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Team_1_Label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>230</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team 1:</string>
   </property>
  </widget>
  <widget class="QComboBox" name="Team_2_ComboBox">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>280</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Team_2_Label">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>290</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Team 2:</string>
   </property>
  </widget>
  <widget class="QLabel" name="Total_Matches_Label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>350</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Total Matches:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="Total_Matches_Entry">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>340</y>
     <width>191</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="Error_Label">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>390</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: #f00;</string>
   </property>
   <property name="text">
    <string>Please fill all fields</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
